sos/plugins/insights.py

```diff
@@ -13,16 +13,16 @@ class RedHatInsights(Plugin, RedHatPlugin):
     '''Collect config and log for Red Hat Insights
     '''
     plugin_name = 'insights'
-    packages = ['redhat-access-insights']
+    packages = ['insights-client']
     profiles = ('system', 'sysmgmt')
-    conf_file = '/etc/redhat-access-insights/redhat-access-insights.conf'
+    conf_file = '/etc/insights-client/insights-client.conf'
```

Comment:
Same comment here: if this applies to e.g. two different major versions of Red Hat Enterprise Linux then both paths need to be handled since the plugin could be used on either.

Reply:
With the updated package old config file is kept as soft link pointing to new config file. 

# ll /etc/redhat-access-insights/redhat-access-insights.conf 
lrwxrwxrwx. 1 root root 41 Sep  5 21:45 /etc/redhat-access-insights/redhat-access-insights.conf -> /etc/insights-client/insights-client.conf

so I thought to collect the new file.